styles.css

```diff
@@ -447,7 +452,7 @@ button:disabled {
   font-weight: normal;
   font-size: 15px;
   color: #979493;
-  width: 600px;
+  width: 588px;
   padding: 14px;
 }
 
```

Comment:
Please use more padding between icons. Kindly see the screenshot below for reference.